src/pages/WorldsDashboard/WorldsDashboard.tsx

```diff
@@ -26,6 +31,35 @@ export const WorldsDashboard: React.FC = () => {
     history,
   ]);
 
+  const user = useUser();
+
+  const worlds = useOwnWorlds(user.userId);
+
+  const hasWorlds = !!worlds.length;
+
+  const renderWelcomePage = useMemo(
+    () => (
+      <>
+        <div className="WorldsDashboard__messages-container">
+          <AdminShowcaseTitle>Let’s create a world</AdminShowcaseTitle>
+          <AdminShowcaseSubTitle>It’s fast and easy</AdminShowcaseSubTitle>
+        </div>
+      </>
```

Comment:
```suggestion
        <div className="WorldsDashboard__messages-container">
          <AdminShowcaseTitle>Let’s create a world</AdminShowcaseTitle>
          <AdminShowcaseSubTitle>It’s fast and easy</AdminShowcaseSubTitle>
        </div>
```

Reply:
Fixed in: https://github.com/sparkletown/sparkle/pull/2371/commits/cef8d195f3cdcfef440c04428ffe7f938bbb620f